### PR TITLE
Fix #21 - Add Padding and icons 

### PR DESCRIPTION
**Pull request by NightmareFox12**
#21 

- Removed local icons
- changed the name of the components folder to _components
- add padding in small screens

### CODE REVIEW

File: packages/nextjs/app/builders/0xD2692F9df925D18D527ABe8b3d99EE9E9C8d75AE/_components/SkillComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
Comment (phipsae) [inline packages/nextjs/app/builders/0xD2692F9df925D18D527ABe8b3d99EE9E9C8d75AE/_components/SkillComponent.tsx:1]: If you're using Next.js, you no longer need to import React manually, as Next.js automatically includes it for you

**Review by phipsae — CHANGES_REQUESTED** [review]
Thanks, @NightmareFox12, for this PR! The code is very neat and clear.

I just have a small comment.

As general advice for future PRs, please make sure to create and check out a new branch. Also, consider providing a more detailed description. You can refer to Zak's contribution guidelines here: https://gist.github.com/ZakGriffith/69d1eb8baebddd7d370b87a65a7e3ec0.

File: packages/nextjs/app/builders/0xD2692F9df925D18D527ABe8b3d99EE9E9C8d75AE/_components/SkillComponent.tsx (hunk 1 above)
Comment (phipsae) [reply packages/nextjs/app/builders/0xD2692F9df925D18D527ABe8b3d99EE9E9C8d75AE/_components/SkillComponent.tsx:1]: @NightmareFox12 don't forget to push the changes :)

**Review by Pabl0cks — APPROVED** [review]
Thanks for the updates @NightmareFox12 !! Looking good to me 🙌

File: packages/nextjs/app/builders/0xD2692F9df925D18D527ABe8b3d99EE9E9C8d75AE/page.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import { Address } from "~~/components/scaffold-eth";
+
+const NightmareFox: NextPage = () => {
+  return (
+    <main className="w-full max-w-6xl mx-auto px-4 sm:px-8 py-12 md:py-16">
+      <div className="grid md:grid-cols-2 gap-8 md:gap-12">
+        <div className="flex flex-col items-center md:items-start gap-6">
+          <div className="rounded-full w-40 h-40 md:w-56 md:h-56 overflow-hidden border-red-500 border-2">
+            <Image
+              src={nightmareFoxImage}
```
[the remark this answers is not in the document]
Comment (NightmareFox12) [reply packages/nextjs/app/builders/0xD2692F9df925D18D527ABe8b3d99EE9E9C8d75AE/page.tsx]: done ! ✨

**Pull request closed without merging**